client/src/components/ElementEditor/Element.js

```diff
@@ -24,10 +32,19 @@ class Element extends PureComponent {
 
 Element.propTypes = {
   element: elementType,
+
 };
 
 Element.defaultProps = {
   element: null,
 };
 
-export default Element;
+export { Element as Component };
+
+export default inject(
+  ['ElementHeader', ],
+  (HeaderComponent,) => ({
+    HeaderComponent,
+  }),
+  () => 'Element'
```

Comment:
I think we could use a more descriptive injector context here - how about `ElementEditor.ElementList.Editor` (since the parents have parts of that already)